package.json

```diff
@@ -31,36 +31,36 @@
   "author": "algoan",
   "license": "ISC",
   "devDependencies": {
-    "@algoan/eslint-config": "^1.1.4",
+    "@algoan/eslint-config": "^1.1.5",
     "@ava/typescript": "^2.0.0",
-    "@commitlint/cli": "^13.2.1",
-    "@commitlint/config-conventional": "^13.2.0",
-    "@semantic-release/git": "^10.0.0",
-    "@types/sinon": "^10.0.6",
-    "@types/uuid": "^8.3.1",
+    "@commitlint/cli": "^17.0.3",
+    "@commitlint/config-conventional": "^17.0.3",
+    "@semantic-release/git": "^10.0.1",
+    "@types/sinon": "^10.0.12",
+    "@types/uuid": "^8.3.4",
     "@yelloan/tslint": "0.3.0",
     "ava": "^3.15.0",
     "commitizen": "^4.2.4",
     "cz-conventional-changelog": "^3.3.0",
-    "eslint": "^8.0.1",
-    "google-pubsub-emulator": "^5.0.0",
-    "husky": "^7.0.4",
-    "lint-staged": "^12.3.4",
+    "eslint": "^8.19.0",
+    "google-pubsub-emulator": "git+https://github.com/ert78gb/google-pubsub-emulator#4aa0ed1b8ef0e8157208384c3f2fc2c8dad1e8b1",
+    "husky": "^8.0.1",
+    "lint-staged": "^13.0.3",
     "nyc": "^15.1.0",
-    "prettier": "^2.5.1",
+    "prettier": "^2.7.1",
     "rimraf": "^3.0.2",
-    "semantic-release": "^18.0.0",
-    "sinon": "^12.0.1",
+    "semantic-release": "^19.0.3",
+    "sinon": "^14.0.0",
     "tslint": "^6.1.3",
-    "typescript": "^4.4.4",
+    "typescript": "^4.7.4",
     "uuid": "^8.3.2"
   },
   "dependencies": {
-    "@google-cloud/pubsub": "^2.18.1",
-    "pino": "^7.4.1"
+    "@google-cloud/pubsub": "^3.0.1",
```

Comment:
this should hopefully fix the security vuln in `google-gax@2` as well, right?

Reply:
yes, it does!